crates/router/src/routes/recon.rs

```diff
@@ -54,7 +54,7 @@ pub async fn get_recon_token(state: web::Data<AppState>, req: HttpRequest) -> Ht
         (),
         |state, user, _, _| recon::generate_recon_token(state, user),
         &authentication::JWTAuth {
-            permission: Permission::MerchantReconWrite,
+            permission: Permission::MerchantAccountRead,
```

Comment:
Can we have separate resource for `ReconToken` and use it here. We can put that resource in both the groups.

If we have this permission here, use should also have `AccountView` group to access this API.